index.html

```diff
@@ -25,27 +29,255 @@
                 <div class="presentation-content">
                     <div class="text-wrapper">
                         <h1 class="text text-primary">I’m Lawrence <br>Glad to see <br> you!</h1>
-                        <p class="text text-secondry">I’m a software developer! I can help you build a product , 
-                            feature or website Look through some of my work and experience! 
+                        <p class="text text-secondry">I’m a software developer! I can help you build a product ,
+                            feature or website Look through some of my work and experience!
                             If you like what you see and have a project you need coded, don’t hestiate to contact me.
                         </p>
-                    
+
                         <div class="social text">
                             <div class="social-header social-items">LET'S CONNECT</div>
                             <div class="social-icons social-items">
-                                <ul id="social-icon-list">
-                                    <li><a href="https://twitter.com/kusilaw"><img src="./images/twitter.svg" alt="twitter"></a></li>
-                                    <li><a href="linkedin.com/in/lawrence-kusi-55a662104"><img src="./images/Linkedin-icon.svg" alt="link"></a></li>
+                                <ul class="social-icon-list">
+                                    <li><a href="https://twitter.com/kusilaw"><img src="./images/twitter.svg"
+                                                alt="twitter"></a></li>
+                                    <li><a href="#"><img src="./images/Linkedin-icon.svg" alt="link"></a></li>
                                     <li><a href="#"><img src="./images/man.svg" alt="man"></a></li>
-                                    <li><a href="https://github.com/kusiLaw"><img src="./images/github.svg" alt="github"></a></li>
+                                    <li><a href="https://github.com/kusiLaw"><img src="./images/github.svg"
+                                                alt="github"></a></li>
                                     <li><a href="#"><img src="./images/like.svg" alt="like"></a></li>
                                 </ul>
                             </div>
                         </div>
                     </div>
-                </div>    
+                </div>
             </section>
+
+            <section class="work">
```

Comment:
The margins for this section are not quite right. The content should be centered and it is slightly slanted to the left. Please fix it.

<img width="227" alt="Screen Shot 2022-07-07 at 9 04 45" src="https://user-images.githubusercontent.com/82426682/177793078-759a9053-a6d6-479a-8e18-40e31956e7a1.png">

<img width="485" alt="Screen Shot 2022-07-07 at 9 05 05" src="https://user-images.githubusercontent.com/82426682/177793130-0b0b1fbc-6990-411b-a8a2-19adc8a0321f.png">